Gems/LmbrCentral/Code/Tests/ShapeTestUtils.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Settings/SettingsRegistryImpl.h>
+#include <AzCore/std/smart_ptr/unique_ptr.h>
+
+namespace AZ
+{
+    class Entity;
+    class Vector3;
+} // namespace AZ
+
+namespace UnitTest
+{
+    class ShapeOffsetFixture
```

Comment:
suggestion: Might be better to not call this Fixture and instead have it as `ShapeOffsetTestsBase` or something like that, just to avoid any confusion with the other main Fixture that gets inherited from (feel free to think of a better name 😅)

Reply:
Renamed.